src/smc-webapp/r_account.cjsx

```diff
@@ -246,49 +242,48 @@ PasswordSetting = rclass
                 {score[result.score]} (crack time: {result.crack_time_display})
             </div>
 
-    render_value : ->
-        switch @state.state
-            when 'view'
-                <Button className='pull-right' onClick={@change_password}  style={marginTop: '8px'}>
-                    Change password
-                </Button>
-            when 'edit', 'saving'
-                <Well style={marginTop:'10px'}>
-                    Current password
-                    <Input
-                        autoFocus
-                        type        = 'password'
-                        ref         = 'old_password'
-                        value       = {@state.old_password}
-                        placeholder = 'Current password'
-                        onChange    = {=>@setState(old_password : @refs.old_password.getValue())}
-                    />
-                    New password
-                    <form onSubmit={(e)=>e.preventDefault();if @is_submittable() then @save_new_password()}>
-                        <Input
-                            type        = 'password'
-                            ref         = 'new_password'
-                            value       = {@state.new_password}
-                            placeholder = 'New password'
-                            onChange    = {=>x=@refs.new_password.getValue(); @setState(zxcvbn:password_score(x), new_password:x)}
-                        />
-                    </form>
-                    {@password_meter()}
-                    <ButtonToolbar>
-                        {@change_button()}
-                        <Button bsStyle='default' onClick={@cancel_editing}>Cancel</Button>
-                    </ButtonToolbar>
-                    {@render_error()}
-                    {@render_saving()}
-                </Well>
+    render_edit : ->
+        <Well style={marginTop:'3ex'}>
+            Current password
+            <Input
+                autoFocus
+                type        = 'password'
+                ref         = 'old_password'
+                value       = {@state.old_password}
+                placeholder = 'Current password'
+                onChange    = {=>@setState(old_password : @refs.old_password.getValue())}
+            />
+            New password
+            <form onSubmit={(e)=>e.preventDefault();if @is_submittable() then @save_new_password()}>
+                <Input
+                    type        = 'password'
+                    ref         = 'new_password'
+                    value       = {@state.new_password}
+                    placeholder = 'New password'
+                    onChange    = {=>x=@refs.new_password.getValue(); @setState(zxcvbn:password_score(x), new_password:x)}
```

Comment:
oh, regarding `zxcvbn`, I see. it assumes to be global. maybe that should be a get_zxcvbn() function and there it does this `require.ensure`.